signer/src/bitcoin/validation.rs

```diff
@@ -989,11 +990,11 @@ impl WithdrawalRequestReport {
         }
 
         let block_wait = bitcoin_chain_tip_height.saturating_sub(self.bitcoin_block_height);
-        if block_wait < WITHDRAWAL_MIN_CONFIRMATIONS {
+        if block_wait < WITHDRAWAL_MIN_CONFIRMATIONS.into() {
```

Comment:
Same here and below, I'd move the `.into()` since we're not dealing with heights anymore and rather a diff.